Tests/Grid/Mapping/Metadata/MetadataTest.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace APY\DataGridBundle\Grid\Tests\Mapping\Metadata;
+
+use APY\DataGridBundle\Grid\Columns;
+use APY\DataGridBundle\Grid\Column\Column;
+use APY\DataGridBundle\Grid\Mapping\Metadata\Metadata;
+use PHPUnit\Framework\TestCase;
+
+class MetadataTest extends TestCase
+{
+    public function testSetFieldsWithString()
+    {
+        $randomField = 'foo_' . rand(0, 100);
+
+        $metadata = new Metadata();
+        $metadata->setFields($randomField);
+
+        $this->assertEquals($randomField, $metadata->getFields());
```

Comment:
As above, don't use getter: duplicate the test and modify this to check argument.